cmd/autoheal/healer.go

```diff
@@ -200,6 +204,47 @@ func (h *Healer) Run(stopCh <-chan struct{}) error {
 	return nil
 }
 
+// Reload all rules in rules cache (by sending "Deleted" + "Added" to queue).
+//
+func (h *Healer) onConfigFilesCanged() {
```

Comment:
I think that using the word _files_ here indicates that this is two low level. Here the healer should not care about the source of the configuration: it may be files, it may be something else. This method should be named `onConfigChanged` and the configuration should have already been reloaded at this point.